docs/source/guides/staking_guide.rst

```diff
@@ -24,126 +45,157 @@ NuCypher Staking Guide
 +-----------------+--------------------------------------------+
 |  ``--index``    | Stake index                                |
 +-----------------+--------------------------------------------+
-|  ``--list``     | List stakes (used with ``stake`` action)   |
-+-----------------+--------------------------------------------+
-|  ``--divide``   | Divide stakes (used with ``stake`` action) |
-+-----------------+--------------------------------------------+
 
 
 
 Interactive Method
 ------------------
 
+*Initialize a new stakeholder*
+
 .. code:: bash
 
-    (nucypher)$ nucypher ursula stake
+    (nucypher)$ nucypher stake new-stakeholder --provider <PROVIDER>  --poa
```

Comment:
Example of a provider needed? (E.g. examples of provider: `ipc:///home/username/.ethereum/goerli/geth.ipc`, `ipc:///tmp/geth.ipc`, <something for infura?>)